tsl/src/bgw_policy/move_hyper_inval_api.c

```diff
@@ -0,0 +1,223 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
+
+#include <postgres.h>
+
+#include "move_hyper_inval_api.h"
+
+#include "bgw/job.h"
+#include "bgw/job_stat.h"
+#include "bgw/timer.h"
+#include "bgw_policy/job.h"
+#include "bgw_policy/job_api.h"
+#include "guc.h"
+#include "hypertable.h"
+#include "jsonb_utils.h"
+#include "policy_config.h"
+#include <utils/elog.h>
+
+#define DEFAULT_MAX_RUNTIME                                                                        \
+	DatumGetIntervalP(DirectFunctionCall3(interval_in, CStringGetDatum("0"), InvalidOid, -1))
+
+Datum
+policy_move_hyper_inval_proc(PG_FUNCTION_ARGS)
+{
+	if (PG_NARGS() != 2 || PG_ARGISNULL(0) || PG_ARGISNULL(1))
+		PG_RETURN_VOID();
+
+	ts_feature_flag_check(FEATURE_POLICY);
+	TS_PREVENT_FUNC_IF_READ_ONLY();
+	policy_move_hyper_inval_execute(PG_GETARG_INT32(0), PG_GETARG_JSONB_P(1));
+
+	PG_RETURN_VOID();
+}
+
+Datum
+policy_move_hyper_inval_check(PG_FUNCTION_ARGS)
```

Comment:
What is this function used for? Looks like it is never executed.

Reply:
Check functions are part of the API and has to be defined. As you noted, there is no test for it. Since these are pretty standard functions, I did not think that was important, but I can add a test for it if you feel strongly about it.

Reply:
Added tests for this.